pallets/proposals/src/mock.rs

```diff
@@ -273,4 +277,4 @@ pub fn build_test_externality() -> sp_io::TestExternalities {
     let mut ext = sp_io::TestExternalities::new(t);
     ext.execute_with(|| System::set_block_number(1));
     ext
-}
\ No newline at end of file
+}
```

Comment:
Maybe there could be a test to sanity check your code. To see if its working as you intend. Or not.